docs/design/first-run-experience-patterns.md

```diff
@@ -0,0 +1,84 @@
+# First-run Experience
+
+A First-run Experience (FRE) is a user's introduction to your add-in. An FRE is presented when a user opens an add-in for the first time and provides them with insight into the functions, features, and/or benefits of the add-in. This experience helps shape the user's impression of an add-in and can strongly influence their likelihood to come back to and continue using your add-in..
```

Comment:
Is the period where the add-in is loading something you consider part of the FRE experience or is that described somewhere else?